### DISCUSSION TITLE
Link Entities

### DISCUSSION THREAD

**Discussion by S1ntez-911** [dmulloy2/ProtocolLib#2203] [Q&A]
I can't find this packet:
![image](https://user-images.githubusercontent.com/91375597/222268579-750d35ed-fd66-499f-8cf4-043d5db4da48.png)
![image](https://user-images.githubusercontent.com/91375597/222268602-3d2292e1-cb1e-4ff1-bbdc-225dd556fb1d.png)

**Comment by lukalt** [comment]
I think this should be `PacketType.Play.Server.MOUNT`
